suite/spdz/src/test/java/dk/alexandra/fresco/suite/spdz/TestStorage.java

```diff
@@ -68,4 +73,71 @@ public void testStoreBigInteger(Storage storage) {
     BigInteger o2 = storage.getObject("test", "key");
     Assert.assertEquals(o1, o2);
   }
+  
+  @Test
+  public void testInitInMemoryStorage() throws Exception {
+    InitializeStorage.initStorage(new Storage[] {new InMemoryStorage()},
```

Comment:
?

Reply:
This tests that nothing goes wrong when initializing the in-memory storage. I tried to use it in a real test, but this proved "hard" - as in: could not easily figure it out within 10 min.

Reply:
I see. In these cases, where it is not clear from the code, it's best to leave a comment on what is actually being tested.